packages/ckeditor5-table/src/utils/table-properties.ts

```diff
@@ -78,6 +78,17 @@ export interface NormalizedDefaultProperties {
 	horizontalAlignment?: string;
 }
 
+/**
+ * Options for the `getNormalizedDefaultProperties` function.
+ */
+type NormalizeTableDefaultPropertiesOptions = {
```

Comment:
This type probably should be exported as the functions are using it and should have it documented.

Reply:
Please also add docs for the fields as described in the original function parameter.

Reply:
Docs adjusted